src/content/documentation/develop/developing-extensions-for-firefox-for-android.md

```diff
@@ -112,6 +114,18 @@ When setting `strict_min_version` in [`browser_specific_settings`](https://devel
 }
 ```
 
+### Android API and UX checklist
+
+Before making your extension publicly available on AMO, test your extension on Android to make sure it works as expected. Reviewers may reject or request changes to submissions that don't meet basic usability requirements.
+
+- Check [Browser Support for JavaScript APIs](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Browser_support_for_JavaScript_APIs) to verify that the APIs your extension uses are available on Firefox for Android.
```

Comment:
Good call out, especially since both `web-ext lint` and the Browser Support for JS APIs page pull compatibility data from https://github.com/mdn/browser-compat-data/.